### PR TITLE
Added Dashboard to User owned resources GET API

### PR DESCRIPTION
**Pull request by jinhyukchang**
### Summary of Changes

Added Dashboard to User owned resources GET API

### Tests

Integration test. Updated unit test

### Documentation

N/A

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [x] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [x] PR includes a summary of changes. 
- [x] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [x] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public functions and the classes in the PR contain docstrings that explain what it does
- [x] PR passes `make test`

### CODE REVIEW

File: metadata_service/api/user.py [hunk 1]
```diff
@@ -163,11 +163,28 @@ def get(self, user_id: str) -> Iterable[Union[Mapping, int, None]]:
...
             resources = self.client.get_table_by_user_relation(user_email=user_id,
                                                                relation_type=UserResourceRel.own)
-            if len(resources['table']) > 0:
-                return {'table': PopularTableSchema(many=True).dump(resources['table']).data}, HTTPStatus.OK
-            return {'table': []}, HTTPStatus.OK
+            LOGGER.info('resources1 : {}'.format(resources))
+            if resources and table_key in resources and len(resources[table_key]) > 0:
+                result[table_key] = PopularTableSchema(many=True).dump(resources[table_key]).data
+
+            LOGGER.info('result1: {}'.format(result))
```
Comment (feng-tao) [inline metadata_service/api/user.py:179]: nitL typo result1 ?

File: metadata_service/api/user.py [hunk 2]
```diff
@@ -163,11 +163,28 @@ def get(self, user_id: str) -> Iterable[Union[Mapping, int, None]]:
...
+                table_key: [],
+                dashboard_key: []
+            }  # type: Dict[str, List[Any]]
+
             resources = self.client.get_table_by_user_relation(user_email=user_id,
                                                                relation_type=UserResourceRel.own)
-            if len(resources['table']) > 0:
-                return {'table': PopularTableSchema(many=True).dump(resources['table']).data}, HTTPStatus.OK
-            return {'table': []}, HTTPStatus.OK
+            LOGGER.info('resources1 : {}'.format(resources))
```
Comment (feng-tao) [inline metadata_service/api/user.py:175]: typo resources1

File: metadata_service/api/user.py (hunk 1 above)
Comment (jinhyukchang) [reply metadata_service/api/user.py:179]: May bad and it was part of debugging. Will remove.

File: metadata_service/api/user.py (hunk 2 above)
Comment (jinhyukchang) [reply metadata_service/api/user.py:175]: May bad and it was part of debugging. Will remove.

**Pull request merged**